crates/edr_napi/Cargo.toml

```diff
@@ -45,11 +43,9 @@ openssl-sys = { version = "0.9.93", features = ["vendored"] }
 napi-build = "2.0.1"
 
 [features]
-tracing = ["edr_evm/tracing", "edr_napi_core/tracing", "edr_provider/tracing"]
+optimism = ["dep:edr_optimism"]
 scenarios = ["rand"]
-
-[profile.release]
```

Comment:
`cargo` was reporting that this setting was ignored. Only the workspace `Cargo.toml`'s profile are respected.